### DISCUSSION TITLE
Assigning a movement

### DISCUSSION THREAD

**Discussion by dragonash2019** [kinsi55/CS_BeatSaber_Camera2#29] [Q&A] [answered]
Was reading the wiki, but I don't see how to assign the test movement script to a camera after placing it in the folder.

**Comment by kinsi55** [comment, marked as answer]
You add it to the appropriate section in the camera config

**Reply by dragonash2019** [reply]
Are you talking about in game or through another json file?

**Reply by kinsi55** [reply]
https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Movement-Scripts

https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Camera-Configuration#movementscript-positionable-only

**Comment by dragonash2019** [comment]
I see, you have to edit the camera script directly and add the new movement script name at the end.
That wasn't entirely clear to me.
I thought once you create the movement script, it had to be selected in game
